packages/core-elements/src/elements/flex-box/flex-box.tsx

```diff
@@ -16,7 +16,10 @@ export interface FlexBoxProps extends Omit<FlexItemProps, 'flex'> {
   rowGap?: Property.RowGap
 }
 
-export interface FlexItemProps extends ContainerProps, DOMAttributes<Element> {
+export interface FlexItemProps
+  extends ContainerProps,
+    DOMAttributes<Element>,
+    HTMLAttributes<Element> {
```

Comment:
```suggestion
export interface FlexItemProps
  extends ContainerProps,
    HTMLAttributes<Element> {
```

HTMLAttributes 타입 자체가 DOMAttributes를 extends 하고 있어서 없어도 될 것 같네요